index.bs

```diff
@@ -1845,21 +1913,23 @@ const e = builder.add(d, c);
 const graph = await builder.build({d, e});
 
 const bufferA = new Float32Array(sizeOfShape(descA.dimensions)).fill(0.5);
-const inputs = {'a': {data: bufferA}};
+const inputs = {'a': new MLTensor(bufferA)};
 
 // Compute both d and e.
-let outputs = await graph.compute(inputs);
+let outputs = graph.compute(inputs);
 console.log(&#96;outputs include ${Object.keys(outputs)}&#96;);
 
 // Compute d.
-outputs = await graph.compute(inputs, {d});
+outputs = graph.compute(inputs, {d});
```

Comment:
For the optional output scenario, user code only needs to specify the required output names. Previous `MLOutput` has all optional fields, so it can be an empty dictionary and not bound to any resources. However with the new `MLTensor`, it looks like user code has to create from a resource which is not necessary for this usage.

Reply:
Yes, in this case `{d}` is a dictionary with a single key associated with a null (`MLTensor`) value. Am I missing something here?

Reply:
I may make mistake for `record<DOMString, MLOutput>` case, it should be `{'d': {}}`. However, for `record<DOMString, MLTensor>`, I am afraid the `{'d': null}` is invalid.

Reply:
As the outputs will be returned by compute, probably we can optimize this usage by allowing user to only specify the output names in an array. e.g.,
```js
outputs = graph.compute(inputs, ['d']);
```

Reply:
We should remove this use case and simplify the API. See more details in my other reply below.

Reply:
Syntax-wise, are you saying that the following is also invalid? 

```javascript
const d = builder.matmul(a, b);
const e = builder.add(d, c);
--> (?) const graph = await builder.build({d, e});
```

Can you tell me exactly how `record<K,V>` should be used and how should we write the above line in question? I've never found a usage sample for it in the WebIDL spec.

Reply:
> ```js
> --> (?) const graph = await builder.build({d, e});
> ```

This code is valid. It is equivalent to `await builder.build({'d': d, 'e': e});`

They are two styles of [object properties definition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions). The former one is so called "shorthand property names" of  ES2015.

The `outputs = graph.compute(inputs, {d});` is invalid is because the object `d` is a type of `Operand` in that example. If `d` is a `MLOutput`, it will be valid.